dashboard/src/actions/repos.ts

```diff
@@ -118,8 +118,9 @@ export const deleteRepo = (
   return async (dispatch, getState) => {
     const {
       clusters: {
+        currentCluster,
         clusters: {
-          default: { currentNamespace },
+          [currentCluster]: { currentNamespace },
```

Comment:
I was surprised I can do this, makes it quite neat (using a reference from the same object destructuring).